### ISSUE TITLE
autophagosome assembly

### ISSUE THREAD

**Issue by ukemi** [geneontology/go-ontology#12122]
@Pauldenny @paolaroncaglia @RLovering @marcfeuermann

Currently all of the autophagosome-related terms are part of macroautophagy. I think this creates a true path violation because autophagosomes are created as part of autophagic cell death and the cvt pathway. Some references refer to the cvt pathway as a 'selective autophagy' pathway (see Rama's ticket on the other tracker).However, it seems like most consider it a related but distinct pathway that is biosynthetic and shares many of the genes and processes of macroautophagy, in particular the formation of an autophagosome. Given that we did not make autophagic cell death a type of autophagy and in the end I think it likely that we decide the cvt pathway is not a type of macroautophagy because it does not have the required catabolic component, I think we need to remove the part_of relations between autophagosome processes and macroautophagy. Perhaps something for discussion on the next conference call. 
Note that doing this will require co-annotation to both processes unless we make a set of specific autophagosome-related processes involved in macroautophagy.

**Comment by ukemi** [comment]
We have decided that the cvt pathway is not a type of autophagy, but shares many of the same gene products. The autophagosome is defined in the ontology as being specific to autophagy. The cvt pathway has a CVT vesicle already defined.

**Comment by paolaroncaglia** [comment]
Following email discussion with David, I edited 'autophagic cell death' to make it has_part 'macroautophagy' rather than 'autophagosome formation'. (I edited the logical def too.) I added a dbxref that illustrates that this change reflects Drosophila biology well.
Here's the revised stanza:

[Term]
id: GO:0048102
name: autophagic cell death
namespace: biological_process
def: "A form of programmed cell death that is accompanied by the formation of autophagosomes. Autophagic cell death is characterized by lack of chromatin condensation and massive vacuolization of the cytoplasm, with little or no uptake by phagocytic cells." [GOC:autophagy, GOC:mah, GOC:mtg_apoptosis, PMID:18846107, PMID:23347517]
comment: The precise nature of autophagic cell death is still being debated, and the link between autophagy and cell death unclear. As autophagy is often induced under conditions of stress that could also lead to cell death, there has been a propagation of the idea that autophagy can act as a cell death mechanism; but others suggest that autophagy may simply be an attempt of dying cells to adapt to lethal stress rather than a mechanism to execute a cell death program. Further studies are required to resolve this controversy (see e.g. PMID:22082964, PMID:22052193, PMID:25236395). In the meantime, curators should carefully examine the experimental evidence presented in papers concerning autophagic cell death, and annotate accordingly. Recently, an instance of autophagic cell death, termed autosis, was discovered that relies on the plasma membrane Na+/K+-ATPase. Autosis was observed in vivo in the brain of rats subjected to an ischemic insult. It's still unclear if all cases of autophagic cell death require the Na+/K+-ATPase or not.
synonym: "autophagic death" BROAD []
synonym: "autosis" RELATED [PMID:25236395]
synonym: "programmed cell death by autophagy" BROAD [GOC:pr]
synonym: "programmed cell death by macroautophagy" EXACT [GOC:cjm]
synonym: "type II programmed cell death" RELATED [GOC:sl]
is_a: GO:0012501 ! programmed cell death
intersection_of: GO:0012501 ! programmed cell death
intersection_of: has_part GO:0016236 ! macroautophagy
relationship: has_part GO:0016236 ! macroautophagy
relationship: only_in_taxon NCBITaxon:2759 {id="GOTAX:0000386"} ! Eukaryota
